### PR TITLE
Added my data

### PR DESCRIPTION
**Pull request by MuhammadHassanRazaa**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -119,6 +120,21 @@ const data = [
 		linkedin: "https://linkedin.com/in/iamKarthikBK",
 		twitter: "https://twitter.com/iamKarthikBK",
 		portfolio: "https://www.nohello.com",
+	},
+	{
+		longitude: 47.54352832810872,
+		latitude: 21.640339625980996,
+		batch: 2020,
+		photo: karthik,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:128]:
```suggestion
		photo: MuhammadHassanRazaa,
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@MuhammadHassanRazaa Your `photo` value is incorrect.

**Comment by MuhammadHassanRazaa** [comment]
Oh, sorry I forgot to change the value.. Done :)

**Review by aravindvnair99 — APPROVED** [review]

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -119,6 +120,21 @@ const data = [
 		linkedin: "https://linkedin.com/in/iamKarthikBK",
 		twitter: "https://twitter.com/iamKarthikBK",
 		portfolio: "https://www.nohello.com",
+	},
+	{
+		longitude: 47.54352832810872,
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:125]: Hey @MuhammadHassanRazaa 
Could you please verify the co-ordinates? Seems to me like you swapped the lat and long by mistake. The location seems to be in UAE currently.

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]

File: src/components/json/worldData.js (hunk 2 above)
Comment (abhinavsri360) [reply src/components/json/worldData.js:125]: I figured this out because your chapter is in Hungary.

Comment (MuhammadHassanRazaa) [reply src/components/json/worldData.js]: Oh yeah, just made the corrections. Thanks for pointing out.

**Review by abhinavsri360 — APPROVED** [review]
Looks good to me.

**Pull request merged**